railties/lib/rails/commands/dbconsole.rb

```diff
@@ -1,13 +1,49 @@
 require 'erb'
 require 'yaml'
 require 'optparse'
+require 'rails/commands/console_helper'
 
 module Rails
   class DBConsole
+    include ConsoleHelper
+
     attr_reader :arguments
 
-    def self.start
-      new.start
+    class << self
+      def parse_arguments(arguments)
```

Comment:
Why this method was moved to class level? I think it is better to keep everything on instance level.